### DISCUSSION TITLE
Custom theme for special parts

### DISCUSSION THREAD

**Discussion by olup** [chakra-ui/chakra-ui#5584] [Q&A]
Hello all,

This must have been asked - but I am mildly accustomed to chakra and I am still having a hard time finding what's the name of the button component's part that can style the icon in a custom theme.

And more generally, how to quickly know what parts are in a component and their naming conventions?

Many thanks

**Comment by olup** [comment]
It occurs to me that the button might not be a multipart component at all - but then how should I go about changing those styles that define the spacing of an icon and the text?

**Reply by nikolovlazar** [reply]
Hi @olup. The `Button` component is a single part component. I understand that the icon part is not explained in the docs, but here's a codesandbox that I created that showcases how you can access the icon part: https://codesandbox.io/s/button-icon-color-mfi93?file=/src/index.tsx

I'll talk with the team about making the `Button` a multi part component, so the Icon part can be styled easily. Thank you for raising this issue.
